repositories/decisions_repository.go

```diff
@@ -71,6 +72,31 @@ func (repo *MarbleDbRepository) DecisionWithRuleExecutionsById(ctx context.Conte
 	return decisions[0], nil
 }
 
+func (repo *MarbleDbRepository) DecisionsByOutcomeAndScore(ctx context.Context, exec Executor,
+	scenarioID string, begin, end time.Time,
+) ([]models.DecisionsByVersionByOutcome, error) {
+	query := NewQueryBuilder().
```

Comment:
really sorry, but this sql query is not correct 🙈 
if you do 
```
FROM decisions
JOIN phantom_decisions USING(scenario_id)
```
you will get millions of rows because it's basically a cartesian product (assuming thousands of decisions and phantom decisions).

What you really want to do is a `UNION ALL`, though really it would be much simpler to just have two separate queries (though they would look almost the same) for the real decisions and the phantom decisions.
Then we get a list of aggregates for the real decisions, a list of aggregates for the phantom decisions, and we can return them together in the dto slice.